lib/streamlit/elements/image.py

```diff
@@ -417,8 +497,9 @@ def marshall_images(
 
             # Following regex allows svg image files to start either via a "<?xml...>" tag eventually followed by a "<svg...>" tag or directly starting with a "<svg>" tag
             if re.search(r"(^\s?(<\?xml[\s\S]*<svg\s)|^\s?<svg\s)", image):
-                proto_img.markup = f"data:image/svg+xml,{image}"
```

Comment:
Changing this from `markup` to `url` might require some additional work:
1) I think this was the only usage of the `markup` property, so we can remove this completely from the proto. 
2) The `markup` triggers additional sanitization for security reasons. We might to check if we need to apply something similar in this situation: 

https://github.com/streamlit/streamlit/blob/eca164ae0a0a5316e6da30312638be8ced2c847f/frontend/src/components/elements/ImageList/ImageList.tsx#L104

Is it required to change `markup` to `url` here, or could we also just pass it to `markup`?

Reply:
I've reopened the PR and now with minimal changes I've left it as an `markup`. I think it might be possible to interpreter it as always url, however `svg` needs to be sanitized with `xssSanitizeSvg`